src/__test__/mountable.test.js

```diff
@@ -20,23 +21,29 @@ describe(`emitMount`, () => {
     document.body.removeChild(node)
   })
 
-  it(`does not emit a mount event if it is not mounted`, () => {
+  it(`does not emit a mount event if it is not mounted`, done => {
     const node = document.createElement(`div`)
 
     // ensure that callback does not get called
-    const callback = () => assert.notOk(true)
+    const callback = sinon.spy()
```

Comment:
Nice catch! Though you don't need sinon here. If you use `done` as the callback, the test will fail if the callback is not called.